packages/cas-ipfs/package.json

```diff
@@ -17,25 +17,25 @@
     "node": ">=10"
   },
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build",
-    "pretest": "docker-compose up -d ipfs mongo",
-    "test": "tsdx test --passWithNoTests",
-    "lint": "tsdx lint src",
-    "lint:fix": "tsdx lint src --fix",
-    "prepare": "tsdx build"
+    "start": "npx tsdx watch",
```

Comment:
hmm, tsdx should be a dev dependency, and prefxing should not be here.

Reply:
Ok I can take out, I was getting tsdx undefined when I tried to run CI github action locally.  Also thought it might imply to install tsdx globally.